test/include/util/transaction_test_util.h

```diff
@@ -193,4 +198,53 @@ class LargeTransactionTestObject {
   // so we don't have to calculate these over and over again
   storage::ProjectedRowInitializer row_initializer_{layout_, StorageTestUtil::ProjectionListAllColumns(layout_)};
 };
+
+class LargeTransactionTestObject::Builder {
+  private:
+    uint16_t max_columns;
+    uint32_t initial_table_size;
+    uint32_t txn_length;
+    std::vector<double> update_select_ratio;
+    storage::BlockStore *block_store;
+    storage::RecordBufferSegmentPool *buffer_pool;
+    std::default_random_engine *generator;
+    bool gc_on;
+    bool bookkeeping;
+    storage::LogManager *log_manager;
+
+    static constexpr uint16_t default_max_columns = 25;
```

Comment:
Can do:
 `uint16_t max_columns = 25;` to do default values and leave the constructor `default`, I believe.